docs/configure-runv-with-containerd-docker.md

```diff
@@ -0,0 +1,57 @@
+# Configure containerd and docker to run with runv
+
+Because ![runv](../README.md) is an [OCI](https://www.opencontainers.org/) compatible runtime, it can work smoothly with other OCI-compatible management tools, namingly [containerd](https://containerd.io/) and [docker](https://www.docker.com/community-edition), by replacing the default runtime [runc](https://github.com/opencontainers/runc).
+
+## Work with containerd
```

Comment:
we need two:
Work with (the newest) containerd
Work with the docker-containerd of the newest docker